script.js

```diff
@@ -0,0 +1,78 @@
+const sumFix = 0.1 + 0.2;
+console.log(`Result - ${sumFix.toFixed(1)}`);
```

Comment:
Be careful `.toFixed()` return the result as a String
Need to be converted back to Number:
```suggestion
console.log(`Result - ${+sumFix.toFixed(1)}`);
```